distributed/bokeh/components.py

```diff
@@ -112,6 +112,8 @@ def __init__(self, n_rectangles=1000, clear_interval='20s', **kwargs):
             export.register_plot(self.root)
             self.root.add_tools(export)
 
+        self.root = row(self.root, name='task_stream', sizing_mode='scale_width')
```

Comment:
@canavandl what was the motivation for this?  Is this just so that we can provide a name?  Is there a nicer way to do this on the original figure?